src/styles/item.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
```

Comment:
Consider using a variable for '58px' since it is used multiple times throughout the code.